core-java-modules/core-java-lang-2/src/test/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingCollectionsUnitTest.java

```diff
@@ -0,0 +1,25 @@
+package com.baeldung.methodmultiplereturnvalues;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public class MultipleReturnValuesUsingCollectionsUnitTest {
+
+    @Test
+    public void whenReturningList_thenMultipleFieldsAreReturned() {
```

Comment:
This is now a `JUnit5` test, which means its class and methods do not need to be `public`.